src/MarketingPageAcceptanceTests.Steps/appsettings.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "hubUrl": "http://host.docker.internal:4444/wd/hub",
+  "hubUrl": "http://localhost:4444/wd/hub",
```

Comment:
why is this not consistent?

Reply:
This has to be localhost for the azure agent to find it (as it has no idea what host.docker.internal resolves to)

Reply:
shouldn't this url be being set in the pipeline, rater than changing the default?

Reply:
I would prefer this to be left as localhost. When running the tests locally it will work on all platforms using localhost, unless the tests are deployed via helm, at which we will know the specific address that the hub can be accessed at.